exonum-java-binding/site-examples/src/test/java/com/exonum/binding/example/testing/TestkitExamplesTest.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2020 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.example.testing;
+
+import static com.exonum.binding.common.crypto.CryptoFunctions.ed25519;
+import static com.exonum.binding.example.testing.IntegrationTestArguments.ARTIFACTS_DIR;
+import static com.exonum.binding.example.testing.IntegrationTestArguments.ARTIFACT_FILENAME;
+import static com.exonum.binding.example.testing.IntegrationTestArguments.ARTIFACT_ID;
+import static com.exonum.binding.fakeservice.FakeService.PUT_TX_ID;
+import static com.exonum.binding.fakeservice.FakeService.RAISE_ERROR_TX_ID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.exonum.binding.common.blockchain.ExecutionStatuses;
+import com.exonum.binding.common.crypto.KeyPair;
+import com.exonum.binding.common.message.TransactionMessage;
+import com.exonum.binding.core.blockchain.Block;
+import com.exonum.binding.core.blockchain.Blockchain;
+import com.exonum.binding.core.blockchain.BlockchainData;
+import com.exonum.binding.core.runtime.ServiceArtifactId;
+import com.exonum.binding.core.storage.indices.MapIndex;
+import com.exonum.binding.fakeservice.FakeSchema;
+import com.exonum.binding.fakeservice.Transactions.PutTransactionArgs;
+import com.exonum.binding.fakeservice.Transactions.RaiseErrorArgs;
+import com.exonum.binding.test.Integration;
+import com.exonum.binding.testkit.FakeTimeProvider;
+import com.exonum.binding.testkit.TestKit;
+import com.exonum.binding.testkit.TestKitExtension;
+import com.exonum.binding.time.TimeSchema;
+import com.exonum.messages.core.runtime.Errors.ExecutionError;
+import com.exonum.messages.core.runtime.Errors.ExecutionStatus;
+import com.google.protobuf.Empty;
+import com.google.protobuf.MessageLite;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+@Integration
+class TestkitExamplesTest {
```

Comment:
They come starting from this sub-section:

https://exonum.com/doc/version/latest/get-started/java-binding/#creating-test-network